packages/dsi-logo-maker/functions/textAnimatedCanvas/index.js

```diff
@@ -1,13 +1,13 @@
-import { getColors } from "../../utils/graphics";
+import { getColors } from '../../utils/graphics';
```

Comment:
I think the only canvas design function missing is `textCanvas`

Reply:
:+1: